Refs #34028 -- Documented that get_script_prefix should not be used outside the request cycle. (#16732)
django/django: docs/ref/urlresolvers.txt

you). In that case, you can call ``get_script_prefix()``, which will return
the script prefix portion of the URL for your Django project. If your Django
project is at the root of its web server, this is always ``"/"``.

.. note::

>     This should not be used outside the request cycle since it relies on values

Reviewer: `request-response cycle` :thinking:

> .. note::
> 
>     This should not be used outside the request-response cycle since it relies
>     on values initialized during the request-response cycle.

Reviewer: I don't like bare "this" :smile:, maybe

```suggestion
.. warning::

    This function **cannot** be used outside of the request-response cycle
    since it relies on values initialized during that cycle.
```

Author: Ah warning is what I was looking for! Yeah this reads better thank you :+1:
